admin/sql/CreateFunctions.sql

```diff
@@ -1166,6 +1166,18 @@ BEGIN
 END;
 $$ LANGUAGE 'plpgsql' STRICT;
 
+CREATE OR REPLACE FUNCTION set_medium_recordings_first_release_dates(medium_ids INTEGER[])
```

Comment:
`set_mediums_recordings_first_release_dates` or `set_media_recordings_first_release_dates`, given the other two above and below are plural and this takes plural IDs? :)